lib/compilers/zig.js

```diff
@@ -132,14 +132,12 @@ export class ZigCompiler extends BaseCompiler {
     }
 
     getIncludeArguments(libraries) {
-        return _.flatten(
-            _.map(libraries, selectedLib => {
-                const foundVersion = this.findLibVersion(selectedLib);
-                if (!foundVersion) return false;
-                // Zig should not have more than 1 path
-                return ['--pkg-begin', foundVersion.name, foundVersion.path, '--pkg-end'];
-            }),
-        );
+        return libraries.flatMap(selectedLib => {
+            const foundVersion = this.findLibVersion(selectedLib);
+            if (!foundVersion) return [];
+            // Zig should not have more than 1 path, but it's still an array so spread it
+            return ['--pkg-begin', foundVersion.name, ...foundVersion.path, '--pkg-end'];
```

Comment:
Need to check in staging that this actually works (Note that the output gets compacted so having path be an array here did not break things, but was still wrong)